po/Makefile.am

```diff
@@ -87,8 +87,8 @@ po/cockpit.manifest.pot:
 		$$(cd $(srcdir) && find $(TRANSLATE) -name 'manifest.json.in')
 
 po/cockpit.appstream.pot:
-	cd $(srcdir) && $(XGETTEXT) --output=$(abs_builddir)/$@ \
-		$$(cd $(srcdir) && find $(TRANSLATE) -name '*.appdata.xml.in')
+	cd $(srcdir) && GETTEXTDATADIRS=$(srcdir)/po $(XGETTEXT) --output=$(abs_builddir)/$@ \
```

Comment:
Why do we need `$GETTEXTDATADIRS` now? That's entirely new.

Reply:
Probably we don't (at least it worked without it) but we use it in the following command, which this one is almost identical with.